README.md

```diff
@@ -19,6 +19,15 @@ Part of the [Equinor Subsurface Visualization project](https://github.com/equino
 Using npm:
 ```bash
 npm install --save @equinor/esv-intersection
+
+To run application in your machine :
+
+$ git clone git@github.com:equinor/esv-intersection
+$ cd esv-intersection
+$ git submodule init
```

Comment:
Even simpler: `git submodule update --init --recursive`

Reply:
yes ... or may be i can add both as it explains better with commands in separate lines.

Reply:
I agree with @nilsml, keep it simple. The less needed from developers to get started, the better. If you want to make it clear, you can put in a comment about what it does instead

Reply:
> We shouldn't have stale PR's. Can you merge this in @ShikhaMishra9?

ok i will make the changes and merge. There was some issue as i was not getting notification from esv-intersection